### ISSUE TITLE
fzp-js

### ISSUE THREAD

**Issue by paulvollmer** [karpawich/fzp2js#1]
just to let you know, it looks like we are building the same library for fritzing fzp files.
https://github.com/fritzing/fzp-js

could be a huge benefit if we can merge/combine the code into one library.

cc @el-j

**Comment by karpawich** [comment]
@paulvollmer Yes, definitely. I emailed you awhile back about combining the code into a single library - I'm glad that you have caught notice! I've also been working on a a general library for converting Fritzing files over to JSON. As of now, I also have an undocumented fzz converter sitting on my computer.

The ultimate goal is to build to make an [Electron-based](https://electronjs.org/) version of Fritzing so that it can be exposed to a wider spectrum of the open-source community. Would you be interested in collaborating on such a project?

**Comment by karpawich** [comment]
@paulvollmer As for the logistics of combining our libraries together, my library is partially a newer version of yours. While they achieve the same end-goal, my changes were too significant to be submitted as a pull request. Thus, I suggest you open a pull-request with any modifications you think would merge the two together.

If you're interested in not just creating this, but a whole library for converting over Fritzing files, then I suggest we create an organization and host it there.

**Comment by karpawich** [comment]
@paulvollmer 

I was originally going to suggest having you open a pull request on my library, but I'll modify the existing one over at **fritzing/fzp-js**.

Let me know if you're interested in developing other fritzing > Javascript projects. I have an undocumented FZZ to JSON converter sitting on my computer, and more projects on the way.

**Comment by paulvollmer** [comment]
i also worked on a fzz javascript implementation and you can find the project here: https://github.com/fritzing/fzz-js

**Comment by karpawich** [comment]
@paulvollmer I've submitted a pull request to **fzp-js** containing the work I've done.

**Comment by karpawich** [comment]
@paulvollmer 

I constructed a single repository which merges all of our libraries together. I suggest that we merge not just our FZP libraries, but *all existing libraries** into one. That way, a single API can be declared for Fritzing in Javascript.

I submitted that repository as [a pull request](https://github.com/fritzing/fritzing-js/pull/5) over at `fritzing-js`. The idea is that we can bring back the React UI you have once a full API is declared with documentation. This way, the UI can be expanded even further to eventually become Fritzing in Javascript.

Also, check out the sweet project logo 😄 

Contact me at [max.karpawich@gmail.com](mailto:max.karpawich@gmail.com) if you want to discuss this in a more extensive capacity.

Thanks!
